### ISSUE TITLE
Commands Not Recognized on Latest VS Code.

### ISSUE THREAD

**Issue by dscalzi** [iCrawl/discord-vscode#179]
VS Code is not recognizing commands entered through the command palette. I have the extension installed, and tried restarting the IDE. VS Code recently added an update where it no longer needs to be reloaded after installing an extension. I'm not sure if this included any breaking changes under the hood.

![image](https://user-images.githubusercontent.com/9703873/52924678-b799c980-32fb-11e9-9c16-5cc5491abcc3.png)

The extension also seems to not be connecting to discord, hence why I tried to manually enable it via the command. Seems to be reflected in the most recent user reviews as well https://marketplace.visualstudio.com/items?itemName=icrawl.discord-vscode#review-details

## VS Code Version Information
Version: 1.31.1 (user setup)
Commit: 1b8e8302e405050205e69b59abb3559592bb9e60
Date: 2019-02-12T02:20:54.427Z
Electron: 3.1.2
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134

## Edit:

The last version (3.2.1) does work, however you need to run the enable command twice. It fails the first time. Latest (3.3.0) not working at all.

**Comment by Aiden01** [comment]
Same here
```
Version: 1.32.0-insider
Commit: a509265116babc17648c9052eeeb8ae2909fed87
Date: 2019-02-13T06:14:01.838Z
Electron: 3.1.3
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.15.0-45-generic
```

**Comment by vvettoretti** [comment]
Same Here

**Comment by vsky84** [comment]
Same here
`Version: 1.31.1 (user setup)
Commit: 1b8e8302e405050205e69b59abb3559592bb9e60
Date: 2019-02-12T02:20:54.427Z
Electron: 3.1.2
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134`

**Comment by FalseData** [comment]
Can confirm on latest.

**Comment by TwanLuttik** [comment]
have the same issue

> Version: 1.31.1 (user setup)
> Commit: 1b8e8302e405050205e69b59abb3559592bb9e60
> Date: 2019-02-12T02:20:54.427Z
> Electron: 3.1.2
> Chrome: 66.0.3359.181
> Node.js: 10.2.0
> V8: 6.6.346.32
> OS: Windows_NT x64 10.0.17763

**Comment by iCrawl** [comment]
Fixed in 3.3.5

**Comment by vvettoretti** [comment]
> Fixed in 3.3.5

Confirm, it works for me, thanks)
